tests/unit/modules/selenium/test_selenium_builder.py

```diff
@@ -224,6 +224,186 @@ def test_chrome_setup_generator(self):
         for idx in range(len(target_lines)):
             self.assertIn(target_lines[idx], content, msg="\n\n%s. %s" % (idx, target_lines[idx]))
 
+    def test_ignore_proxy_option_generator(self):
+        self.configure({
+            "execution": [{
+                "scenario": "loc_sc"}],
+            "scenarios": {
+                "loc_sc": {
+                    "requests": [{
+                        "url": "bla.com"}]}},
+            "modules": {
+                "selenium": {
+                    "options": {
+                        "ignore_proxy": True}}}})
+
+        self.obj.prepare()
+        with open(self.obj.script) as fds:
+            content = fds.read()
+
+        target = "options.ignore_local_proxy_environment_variables()"
```

Comment:
check that this `if webdriver.__version__.split('.')[0]) > 3:` exists in python file